docs/articles/baseplatform/creating-sdk.rst

```diff
@@ -1,5 +1,7 @@
 :orphan:
 
+.. _building-the-sdk:
```

Comment:
What is this used for?

Reply:
Valid point! It will be referenced from (for example) the PELUX Baseline SWF. Perhaps we should try to make sure we have labels for all major headings in the blueprint to make it possible to reference them in other docs?

Reply:
Isn't it enough to use the path and name of the target document for creating hyperlinks?

Reply:
That would be possible, but one wants to link to a specific section of a file using <pre>:ref:\`reference-id\`</pre> style references I think.

Reply:
Ah, ok.